### PR TITLE
issue33: add "help" button

### LINKED ISSUE #33: Add a "help" button

### PR DESCRIPTION
**Pull request by ShlomiShitrit**
fix #33

### CODE REVIEW

File: src/app/components/CSVButton/CSVButton.tsx [hunk 1]
```diff
@@ -12,13 +12,10 @@ export default function CSVButton({ queryWords }: { queryWords: WordData[] }) {
...
-        <Button
-            variant="contained"
-            color="primary"
+        <CustomButton
             onClick={handleDownload}
             sx={styles.button}
-        >
-            Download Results
-        </Button>
+            text="Download Results"
```
Comment (nadavWeisler) [inline src/app/components/CSVButton/CSVButton.tsx]: Resources

File: src/app/components/ChatButton/ChatButton.tsx [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+import { Grid } from "@mui/material";
+import CustomButton from "@/app/components/CustomButton";
 import SendIcon from "@mui/icons-material/Send";
 
 export default function ChatButton() {
     return (
         <Grid item xs={2}>
-            <Button
+            <CustomButton
                 sx={{ width: "120%" }}
```
Comment (nadavWeisler) [inline src/app/components/ChatButton/ChatButton.tsx]: To style

File: src/app/components/ChatButton/ChatButton.tsx [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
-                color="primary"
-                variant="contained"
                 endIcon={<SendIcon />}
                 type="submit"
-            >
-                Send
-            </Button>
+                text="Send"
```
Comment (nadavWeisler) [inline src/app/components/ChatButton/ChatButton.tsx]: Resources

File: src/app/components/CSVButton/CSVButton.tsx (hunk 1 above)
Comment (ShlomiShitrit) [reply src/app/components/CSVButton/CSVButton.tsx]: Done

File: src/app/components/ChatButton/ChatButton.tsx (hunk 2 above)
Comment (ShlomiShitrit) [reply src/app/components/ChatButton/ChatButton.tsx]: Done

File: src/app/components/ChatButton/ChatButton.tsx (hunk 3 above)
Comment (ShlomiShitrit) [reply src/app/components/ChatButton/ChatButton.tsx]: Done

**Review by nadavWeisler — APPROVED** [review]

**Pull request merged**